nmartinyuste/code-test-karumi
# feat(login): add login login (closes #10) and persistance (closes #11)

## Description
Added auth service.
Added api service.
Added non auth && auth guards.
Added cors.
Added home page with logout button.

## Closes issue(s)
#10 #11 

## How to test / repro
npm run code-test-karumi in root project and open localhost:4200.
Login with correct users, check persistance and logout.

## Changes include


## Checklist
